tests/test_comfy_api.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+from comfy_api_simplified.comfy_api_wrapper import ComfyApiWrapper
+from comfy_api_simplified.comfy_workflow_wrapper import ComfyWorkflowWrapper
+from PIL import Image, ImageChops
+import pytest_asyncio
+
+import io
+import os
+
+
+from .conftest import test_data_dir
+
+
+def compare_images(img1, img2):
+    return ImageChops.difference(img1, img2).getbbox() is None
+
+
+def load_image_from_test_data(image_name):
+    path = os.path.join(test_data_dir, image_name)
+    return Image.open(path)
+
+
+def test_upload_image(api_wrapper: ComfyApiWrapper):
+    path = os.path.join(test_data_dir, "input.png")
+    assert os.path.exists(path)
+    try:
+        response = api_wrapper.upload_image(path)
+    except:
+        pytest.fail("Unexpected error")
+
+    assert response
+
+
+def test_queue_prompt(
+    api_wrapper: ComfyApiWrapper, workflow_wrapper: ComfyWorkflowWrapper
+):
+    api_wrapper.upload_image(os.path.join(test_data_dir, "input.png"))
+
+    response = api_wrapper.queue_prompt(workflow_wrapper)
+    prompt_id = response["prompt_id"]
+    assert api_wrapper.get_queue_size_before(prompt_id) == 0
+
+
+@pytest.mark.asyncio
+async def test_get_history(
+    api_wrapper: ComfyApiWrapper, workflow_wrapper: ComfyWorkflowWrapper
+):
+    api_wrapper.upload_image(os.path.join(test_data_dir, "input.png"))
```

Comment:
Didn't run on fresh comfyui instance without explicit image parameters set
```suggestion
    image_metadata = api_wrapper.upload_image(os.path.join(test_data_dir, "input.png"))
    workflow_wrapper.set_node_param("Load Image", "image", f"{image_metadata['subfolder']}/{image_metadata['name']}")
```

Reply:
Indeed, there was an issue, fixed on the last commit, please have a look.